_projects/2023/100061715/100061715.Rmd

```diff
@@ -0,0 +1,256 @@
+---
+title: "Unveiling birth clinical practice in Spain: mapping the cesarian sections and birth dynamics"
+description: "Exploring the nuances of maternal healthcare in Spain through an augmented map, showcasing original cesarean section rates alongside additional insights such as total births per autonomous community and cesarean rates adjusted for the average maternal age."
+categories: 2023
+author: "Celia Muñoz"
+date: "`r Sys.Date()`"
+---
+
+```{r}
+# Loading packages
+library(tidyverse)
+library(mapSpain)
+library(readr)
+library(ggplot2)
+library(sf)
+library(stringdist)
+library(fuzzyjoin)
+library(ggtext)
+```
+
+
+```{r}
+# Adjusting locale settings
+my_locale <- locale(decimal_mark = ",", grouping_mark = ".")
+
+
+# Reading data
+cesareas <- read_delim("tasa_cesareas.csv", 
+                 skip = 10, 
+                 delim = ";",
+                 col_names = c("ccaa", "n_partos", "estancia_partos", "edad_partos", "n_cesareas", "edad_cesareas", "estancia_cesareas", "tasa_cesareas"),
+                 locale = my_locale)
+
+# Cleaning data and generating a new cesarea rate
+cesareas <- cesareas %>% 
+  na.omit() %>% 
+  filter(ccaa != "Total Altas") %>% 
+  mutate(tasa_cesareas = (n_cesareas / n_partos) * 100)
+
+ccaa <- cesareas %>% 
+  select(ccaa)
+
+```
+```{r}
+# Loading map data
+spain <- esp_get_ccaa()
+
+# Ploting the basic map
+ggplot(spain) +
+  geom_sf()
+```
+
+
+
+```{r}
+# Comprobamos que los nombres de las CCAA coinciden y hacemos ajustes para mejorar el join
+cesareas <- cesareas %>% 
+  stringdist_join(
+    spain,
+    by = c("ccaa" = "ine.ccaa.name"),
+    method = "jaccard",
+    mode = "left",
+    max_dist = 0.3
+  ) %>% 
+  mutate(
+    ine.ccaa.name = if_else(ccaa == "Rioja (La)", "Rioja, La", ine.ccaa.name)) %>% 
+  select(ccaa, ine.ccaa.name, n_partos, edad_partos, tasa_cesareas, edad_cesareas) # Keeping the main variables
+
+# Joining map data and cesarean data
+cesareas_map <- spain %>% 
+  left_join(
+    cesareas,
+    by = "ine.ccaa.name"
+  )  
+
+summary(cesareas_map)
```

Comment:
This summary is very long and not very informative. If you want just to show the resulting columns, it's best to use `str(cesareas_map)`.